largest_sum_contiguous_subarray/LargestSumContiguousSubarray.c

```diff
@@ -0,0 +1,26 @@
+#include<stdio.h>
+#include<stdlib.h>
+
+int LargestSumContinousSubArray(int a[], int size) {
```

Comment:
Make it `const int arr*` variables name should make sense a little bit as far as possible.